operator/cmd/mesh/testdata/manifest-generate/data-snapshot/charts/istio-operator/templates/deployment.yaml

```diff
@@ -30,7 +30,7 @@ spec:
               memory: 128Mi
           env:
             - name: WATCH_NAMESPACE
-              value: {{.Values.istioNamespace}}
```

Comment:
We need to maintain backwards compatibility by marking deprecated for a release.

Reply:
@ostromart Moved the `istioNamespace` back and added deprecated message, also use name values of `watchedNamespaces` if it exists, otherwise use `istioNamespace`.

Reply:
This LGTM.